src/plugins/wizard/public/application/components/searchable_dropdown.scss

```diff
@@ -0,0 +1,26 @@
+@import "../variables";
+
+.searchableDropdown {
+  overflow: "hidden";
+}
+
+.searchableDropdown .euiPopover,
+.searchableDropdown .euiPopover__anchor {
+  width: 100%;
+}
+
+.searchableDropdown--fixedWidthChild {
+  width: calc(#{$wizSideNavWidth} - #{$euiSizeXL} * 2) ;
+}
+
+.searchableDropdown--topDisplay {
+  padding-right: $euiSizeL;
+}
+
+
+.searchableDropdown--selectableWrapper .euiSelectableList {
+  // When clicking on the selectable content it will "highlight" itself with a box shadow
+  // This turns that off
+  box-shadow: none !important;
+  margin: $osdDropdownNegativePadding - 4;
```

Comment:
you seem to be only using this value once. Since you have already subtracted 1 from the original value, is this math also needed?